### PR TITLE
docs: updated doc on splash screen

### PR DESCRIPTION
**Pull request by Xiaoy312**
GitHub Issue (If applicable): resolved: #5507

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the new behavior?
- minor changes to splash screen docs and updated it according to the new how-to template

## Other information
related sample is in another PR: unoplatform/Uno.Samples#58

### CODE REVIEW

File: doc/articles/splash-screen.md [hunk 1]
```diff
@@ -2,89 +2,179 @@
...
 
-## WebAssembly
+## Step 2 - UWP
 
-1. In the Wasm project head, navigate to `WasmScripts/AppManifest.js` 
+1. In the `.UWP` project, delete the old splash screen file `SplashScreen.scale-200.png` under the `Asset\` folder.
 
-2. Add your splash screen image
+    > [!NOTE]
+    > Do not confused this with the one from `.Shared` project.
```
Comment (davidjohnoliver) [inline doc/articles/splash-screen.md:80]:
```suggestion
    > Do not confuse this with the one from `.Shared` project.
```

File: doc/articles/splash-screen.md [hunk 2]
```diff
@@ -2,89 +2,179 @@
...
 
-``` javascript
-var UnoAppManifest = {
-splashScreenImage: “Assets/SplashScreen.scale-200.png”,
-splashScreenColor: “#101010”,
-displayName: “Uno.SplashScreenEverywhere”
-}
-```
-Note: Currently, you need to set an explicit scale of the splash screen image.
+1. open the `Package.appxmanifest` and navigate to `Visual Assets > SplashScreen`.
```
Comment (davidjohnoliver) [inline doc/articles/splash-screen.md:82]:
```suggestion
1. Open the `Package.appxmanifest` and navigate to `Visual Assets > SplashScreen`.
```

File: doc/articles/splash-screen.md [hunk 3]
```diff
@@ -2,89 +2,179 @@
...
+1. Add your image path to the `Image View`
+
+    ``` xml
+    <imageView ... image="Assets/SplashScreen">
+    ```
+
+1. Open to `info.plist` and update the `UILaunchStoryboardName` value to `LaunchScreen`.
+
+    > [!TIP]
+    > iOS caches the splash screen to improve the launch time, even across re-installs.In order to see the actual changes made, you need to restart the iPhone or simulator. Alternatively, you rename the `CFBundleIdentifier` in `info.plist` incrementally (eg: MyApp1 -> MyApp2) before each build.
```
Comment (davidjohnoliver) [inline doc/articles/splash-screen.md:160]: 👍 

```suggestion
    > iOS caches the splash screen to improve the launch time, even across re-installs. In order to see the actual changes made, you need to restart the iPhone or simulator. Alternatively, you can rename the `CFBundleIdentifier` in `info.plist` incrementally (eg: MyApp1 -> MyApp2) before each build.
```

**Review by davidjohnoliver — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

File: doc/articles/splash-screen.md [hunk 4]
```diff
@@ -2,89 +2,179 @@
 
 This article covers how to add a splash screen to your application.
 
-As a first step, navigate to `YourProject.Shared > Assets` and add all [scales](#table-of-scales) of your splash screen image.
+> [!TIP]
+> The complete source code that goes along with this guide is available in the [unoplatform/Uno.Samples](https://github.com/unoplatform/Uno.Samples) GitHub repository - [SplashScreenSample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/SplashScreenSample)
 
-## UWP
+## Prerequisites
```
Comment (jeromelaban) [inline doc/articles/splash-screen.md:8]: I think we should redirect to the common prerequisites documentation, it may get outdated quickly.

**Review by carldebilly — APPROVED** [review]

**Pull request merged**